src/coreclr/vm/peassembly.cpp

```diff
@@ -665,6 +665,7 @@ PEAssembly::PEAssembly(
     m_isSystem = isSystem;
     m_pHostAssembly = nullptr;
     m_pFallbackBinder = nullptr;
```

Comment:
Is it possible to delete `m_pFallbackBinder` field and implement `GetFallbackBinder` as:
```
    PTR_AssemblyBinder GetFallbackBinder()
    {
        LIMITED_METHOD_CONTRACT;

        // Fallback binder is only used when there is no host assembly
        return (m_pHostAssembly != NULL) ? NULL : m_pAssemblyBinder;
    }
```
?